src/Fixer/Phpdoc/GeneralPhpdocAnnotationRemoveFixer.php

```diff
@@ -105,7 +104,6 @@ protected function createConfigurationDefinition()
         $annotations = new FixerOptionBuilder('annotations', 'List of annotations to remove, e.g. `["author"]`.');
         $annotations = $annotations
             ->setAllowedTypes(array('array'))
-            ->setDefault(array())
```

Comment:
PSR0 has a hard requirement on the `dir` as well I think. But sure I can revert this and adjust the test if needed.